.github/workflows/publish-snapshot.yml

```diff
@@ -1,48 +1,75 @@
-name: Snapshot builds
-
+# GitHub Actions workflow to automatically publish snapshot builds.
+name: "Snapshot"
```

Comment:
Currently it's:

```
Snapshot / Publish
Snapshot / Docker publish nightly
```

what do you think about:

```
Publish snapshots / Maven
Publish snapshots / Docker
```

Reply:
I think that would be better :)